contributors.md

```diff
@@ -28,7 +28,7 @@ If you would like to use use a visual editor, copy and paste the table below her
 | Beth Signal          | Yes        |                           |          |                |             |
 | Paul Brodersen       | Yes        |                           |          |                |             |
 | Marc G. Chevrette    | Yes        |                           | Yes      |                |             |
-| Ben Lengerich        | Yes        |                           |          |                |             |
+| Ben Lengerich        | Yes        |                           | Yes         |                |             |
```

Comment:
You could remove some space here to keep the `|` column separators aligned.  It's only stylistic though, the table won't be affected.